MediaGalleryCatalogUi/Test/Mftf/ActionGroup/AdminMediaGalleryAddImageToProductDescriptionActionGroup.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminMediaGalleryAddImageToProductDescriptionActionGroup">
```

Comment:
Are we using this Action Group? Didn't find it's usage.

Reply:
removed